src/background/providers/hcaptcha.ts

```diff
@@ -1,19 +1,76 @@
+import * as voprf from '../voprf';
```

Comment:
there should be a test file associated to this change, similar to `cloudflare.test.ts`

Reply:
added a basic test file in this path `src/background/providers/hcaptcha.test.ts` 👍